crates/uv/src/commands/tool/list.rs

```diff
@@ -40,6 +40,15 @@ pub(crate) async fn list(
     }
 
     for (name, tool) in tools {
+        // Skip invalid tools
+        let Ok(tool) = tool else {
+            warn_user!(
+                "Ignoring malformed tool `{name}` (use {} to remove)",
```

Comment:
```suggestion
                "Ignoring malformed tool `{name}` (use `{}` to remove)",
```

Reply:
It's green though so I intentionally omitted this, what do you think?

Reply:
Oh. I don't think the color is enough alone (and it's not great for users with colorblindness or colors turned off).